Content.Server/VendingMachines/VendingMachineSystem.cs

```diff
@@ -435,6 +437,10 @@ public override void Update(float frameTime)
                     }
                 }
             }
+            foreach (var (comp, _) in EntityQuery<VendingMachineComponent, EmpDisabledComponent>())
```

Comment:
EntityQuery should always be done from least common component to most common as this is how the engine does it internally (at least until we get archetypes in the distant future).

Also if you use EntityQueryEnumerator there's an overload with the component .Owner